lib/ansible/module_utils/openstack.py

```diff
@@ -114,16 +114,22 @@ def openstack_cloud_from_module(module, min_version='0.12.0'):
         # Due to the name shadowing we should import other way
         import importlib
         sdk = importlib.import_module('openstack')
+        sdk_version = importlib.import_module('openstack.version')
     except ImportError:
         module.fail_json(msg='openstacksdk is required for this module')
 
     if min_version:
-        if StrictVersion(sdk.version.__version__) < StrictVersion(min_version):
+        if StrictVersion(sdk_version.__version__) < StrictVersion(min_version):
             module.fail_json(
-                msg="To utilize this module, the installed version of"
+                msg="To utilize this module, the installed version of "
                     "the openstacksdk library MUST be >={min_version}".format(
                         min_version=min_version))
 
+    if StrictVersion(sdk_version.__version__) < StrictVersion('0.10.0'):
+        module.fail_json(
+            msg="To utilize this module, the installed version of "
+                "the openstacksdk library MUST be >=0.10.0")
+
```

Comment:
Why this code change is required?
Isn't this already checked in the lines above?

Reply:
`min_version` also defaults to `0.12.0`

Reply:
Another version could be used.

Reply:
When openstacksdk 0.9.19 is used (assuming caller is using a lower version than the default one), without the additional check, the following exception occurs:

```
Traceback (most recent call last):
  File "~/.ansible/tmp/ansible-tmp-1555747400.49-76880204403181/AnsiballZ_os_security_group.py", line 113, in <module>
    _ansiballz_main()
  File "~/.ansible/tmp/ansible-tmp-1555747400.49-76880204403181/AnsiballZ_os_security_group.py", line 105, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "~/.ansible/tmp/ansible-tmp-1555747400.49-76880204403181/AnsiballZ_os_security_group.py", line 48, in invoke_module
    imp.load_module('__main__', mod, module, MOD_DESC)
  File "/tmp/ansible_os_security_group_payload_0EWiaz/__main__.py", line 163, in <module>
  File "/tmp/ansible_os_security_group_payload_0EWiaz/__main__.py", line 115, in main
  File "/tmp/ansible_os_security_group_payload_0EWiaz/ansible_os_security_group_payload.zip/ansible/module_utils/openstack.py", line 162, in openstack_cloud_from_module
AttributeError: 'module' object has no attribute 'exceptions'
```

The current code isn't compatible with openstacksdk < 0.10.0.
Because this is a `module_utils`, other module not belonging to the ansible repository might use this method with another value than the default one.

Reply:
@marsavela @dominikholler: please, could you take a look at this ?